docs/usage.md

```diff
@@ -383,17 +383,21 @@ Turn this on to utilize BWA Mem instead of `bwa aln` for alignment. Can be quite
 
 Users can configure to keep/discard/extract certain groups of reads efficiently in the nf-core/eager pipeline. 
 
-### `--bam_keep_mapped_only`
+### `--bam_analyse_mapped_only`
 
-This can be used to only keep mapped reads for downstream analysis. By default turned off, all reads are kept in the BAM file. Unmapped reads are stored both in BAM and FastQ format e.g. for different downstream processing.
+This can be used to only keep mapped reads in the BAM file for downstream analysis. By default turned off, where all reads are kept in the bam file. Unmapped reads are stored both in BAM and FastQ format e.g. for different downstream processing.
+
+### `--bam_discard_unmapped_entirely`
+
+This discards all unmapped and extracted reads entirely. By default, this is turned off. 
 
 ### `--bam_keep_all`
```

Comment:
What is the difference of this flag verses `--bam_retain_all_reads`?

Reply:
`--bam_discard_unmapped_entirely` 

Removes the unmapped reads file only, the BAM file contains only mapped reads and unmapped reads are entirely discarded (no fastq/bam at all present).


I guess I'll have to give this another proper thought. Right now bam_retain_all_reads just is the thing to turn on BAM filtering in general.